### PR TITLE
Chapter 06 examples

### PR DESCRIPTION
**Pull request by villares**

### CODE REVIEW

File: chp06_agents/NOC_06_01_Seek/NOC_06_01_Seek.pyde [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# The Nature of Code
+# Daniel Shiffman
+# http:#natureofcode.com
+
+# Seeking "vehicle" follows the mouse position
+
+# Implements Craig Reynold's autonomous steering behaviors
+# One vehicle "seeks"
+# See: http:#www.red3d.com/cwr/
```
Comment (jdf) [inline chp06_agents/NOC_06_01_Seek/NOC_06_01_Seek.pyde]: //

File: chp06_agents/NOC_06_01_Seek/NOC_06_01_Seek.pyde [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+from Vehicle import Vehicle
```
Comment (jdf) [inline chp06_agents/NOC_06_01_Seek/NOC_06_01_Seek.pyde]: In general, python modules (and therefore the implementation file names) begin with a lower-case letter.

File: chp06_agents/NOC_06_01_Seek/Vehicle.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# The Nature of Code
+# Daniel Shiffman
+# http:#natureofcode.com
+
+# The "Vehicle" class
+
+class Vehicle():
+    def __init__(self,x, y):
```
Comment (jdf) [inline chp06_agents/NOC_06_01_Seek/Vehicle.py]: Auto-format this and all files.

File: chp06_agents/NOC_06_01_Seek/Vehicle.py [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+        # A vector pointing from the location to the target
+        desired = target - self.position
+
+        # Scale to maximum speed
+        desired.setMag(self.maxspeed)
+
+        # Steering = Desired minus velocity
+        steer = desired - self.velocity
+        steer.limit(self.maxforce)  # Limit to maximum steering force
```
Comment (jdf) [inline chp06_agents/NOC_06_01_Seek/Vehicle.py]: This comment doesn't add anything.

**Review by jdf — COMMENTED** [review]
Thanks for doing this!

File: chp06_agents/NOC_06_01_Seek/Vehicle.py (hunk 3 above)
Comment (villares) [reply chp06_agents/NOC_06_01_Seek/Vehicle.py]: Sorry, will do that.

**Comment by jdf** [comment]
I see you're making chunks of changes; just send me a comment when you're ready for further review.

**Comment by villares** [comment]
Thanks, I think those committed are OK now, next week I might be able to add more examples.

File: chp07_CA/Exercise_7_01_WolframCA_randomizedrules/CA.py [hunk 5]
```diff
@@ -0,0 +1,92 @@
+# The Nature of Code
+# Daniel Shiffman
+# http://natureofcode.com
+
+# Wolfram Cellular Automata
```
Comment (jdf) [inline chp07_CA/Exercise_7_01_WolframCA_randomizedrules/CA.py:5]: I thought this pull request was about chapter 6 examples. Aren't these chapter 7 examples in another pull request?

**Comment by villares** [comment]
This is probably my git incompetence showing, sorry. (Confused by branches).

**Comment by villares** [comment]
Look, I have no idea how to work on branches and stuff... I have tried to fix/remove the `import *` issue again. Please bail me out so I can start fresh.

**Comment by jdf** [comment]
I don't know what you mean by "bail you out".

**Comment by villares** [comment]
Means help me sort my multiple branch/pull request (self inflicted) nightmare.

File: chp06_agents/NOC_06_01_Seek/NOC_06_01_Seek.pyde (hunk 1 above)
Comment (villares) [reply chp06_agents/NOC_06_01_Seek/NOC_06_01_Seek.pyde]: fixed!

File: chp06_agents/NOC_06_01_Seek/NOC_06_01_Seek.pyde (hunk 2 above)
Comment (villares) [reply chp06_agents/NOC_06_01_Seek/NOC_06_01_Seek.pyde]: I think I have fixed this, I'm not sure (branching nightmare)

**Comment by jdf** [comment]
Looks good to me.

**Pull request merged**